src/test/java/core/basesyntax/FruitDaoServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.serviceimpl.FruitDaoServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitDaoServiceTest {
+    private static FruitDaoServiceImpl fruitDaoService;
+    private static Storage storage;
+    private static List<FruitTransaction> transactions;
+
+    @Before
+    public void setUp() {
+        fruitDaoService = new FruitDaoServiceImpl();
+        transactions = new ArrayList<>();
+        transactions.add(new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                "apple",10));
+        transactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                "bananas",120));
+        Storage.fruitTransactions.clear();
```

Comment:
It's better to clear your test data from storage after tests, not before tests
![image](https://user-images.githubusercontent.com/117547096/215285874-ccb15b48-cd16-4731-bdd9-87902a06c491.png)